TravelPoints-BackEnd/src/main/java/com/example/travelpointsbackend/service/implementation/UserWishlistServiceImplementation.java

```diff
@@ -121,9 +133,24 @@ public UserWishlistAndAttractionDTO getUserWishlistAndAttraction(Long userId) {
                 return userWishlistDTO;
 
             }
-            return null;
+            else{
+                UserWishlistAndAttractionDTO userWishlistDTO = new UserWishlistAndAttractionDTO();
+                userWishlistDTO.setUserId(currentUser.getUserId());
+                userWishlistDTO.setAddedAt(null);
+                userWishlistDTO.setWishlistId(null);
+                userWishlistDTO.setAttractionDetailDTOList(new ArrayList<>());
+                return userWishlistDTO;
+            }
+        }
+        else{
+            UserWishlistAndAttractionDTO userWishlistDTO = new UserWishlistAndAttractionDTO();
+            userWishlistDTO.setUserId(null);
+            userWishlistDTO.setAddedAt(null);
+            userWishlistDTO.setWishlistId(null);
```

Comment:
these are null by default when creating a new object, unless you explicitly initializing them in the constructor